src/cli.ts

```diff
@@ -263,10 +263,10 @@ async function buildAndSendTxUsingPrivateKey(
 //////////////////////////////////////////////////////////////////////////////////////////
 // initializing ctx.json
 
-export async function initCtxJsonFromOptions(options: OptionValues): Promise<void> {
+export async function initCtxJsonFromOptions(options: OptionValues, derivationPath = DERIVATION_PATH): Promise<void> {

```

Comment:
This particular function is coming from the flare repository and I've only added a param so I could reuse it. Shall I add documentation for it or leave it be since other functions from the flare repository are also like this?